src/Utility/Test/TestTriangleQuadrature.C

```diff
@@ -0,0 +1,33 @@
+//==============================================================================
+//!
+//! \file TestTriangleQuadrature.C
+//!
+//! \date Jul 31 2019
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Tests for triangle quadratures.
+//!
+//==============================================================================
+
+#include "TriangleQuadrature.h"
+
+#include "gtest/gtest.h"
+#include <numeric>
+
+
+TEST(TestTriangleQuadrature, Sanity)
+{
+  EXPECT_EQ(TriangleQuadrature::getCoord(-1), nullptr);
+  EXPECT_EQ(TriangleQuadrature::getWeight(-1), nullptr);
```

Comment:
Doesn't it make more sense to check getCoord(2)/getWeigth(2) instead, as there is no two-point rule?

Reply:
i can do both.